src/loaders/common/loader_helper.py

```diff
@@ -586,3 +586,43 @@ def dump_json_to_file(json_file_path: str | Path, json_data: dict[str, Any] | li
     """
     with open(json_file_path, "w", encoding="utf8") as json_file:
         json.dump(json_data, json_file, indent=2)
+
+
+def get_meta_file_path(
+        source_dir: Path | str,
+        structured_upa: str
+) -> Path:
+    """
+    Get the metadata file path for a specific workspace id under sourcedata/WS/<env>.
+
+    source_dir - The directory for a specific workspace id under sourcedata/WS/<env>
+    structured_upa - The UPA of a workspace object in the format of "<wsid>_<objid>_<ver>"
+    """
+    source_dir = Path(source_dir)
+    metafile = source_dir / structured_upa / f"{structured_upa}.meta"
```

Comment:
This seems only work if `source_dir` is type Path?

Reply:
I cast it to Path in case if it's not.

Reply:
I inline the casting. I think that makes more sense.

Reply:
👍